src/modules/keycloak-administration/domain/user.ts

```diff
@@ -25,7 +27,9 @@ export class User<WasPersisted extends boolean> {
         email: string | undefined,
         createdDate: Date,
         externalSystemIDs: ExternalSystemIDs,
+        enabled: boolean,
+        attributes: Record<string, string>,
     ): User<WasPersisted> {
-        return new User(id, username, email, createdDate, externalSystemIDs);
+        return new User(id, username, email, createdDate, externalSystemIDs, enabled, attributes);
```

Comment:
externalSystemIDs also use the keycloak attributes. Just fyi, since we may need to separate these two.
Maybe it would be better to explicitly have a "disabledReason" and not the entire attributes?

Reply:
I added a more specific type here.